### ISSUE TITLE
Database not necessary

### ISSUE THREAD

**Issue by marcofucci** [uktrade/export-wins-ui#5]
The UI does not need to have a database so it would be better to delete it and refactor the code so that the logic around users do not use it

**Comment by danielquinn** [comment]
I thought about this, but there's still the standard Django requirements and Django's user model which are all db dependent.  I'd love to make it independent of a database, but I'm afraid I don't know how to drop all of these.

**Comment by danielquinn** [comment]
This is turning out to be a bigger pain in the ass than I originally thought.  Currently, user sessions are stored in the database (it's the default and it's easy) but I hear we can move that to Redis -- not something I've done before though.

There's a whole bunch of other stuff that Django assumes is there by default too, sites framework contenttypes etc.  I know I'm not personally using any of these features directly, but I don't know if there are other dependencies (such as Django itself) that expect that stuff to be around.

TL;DR I agree this needs to be done, but it's not as easy as changing a config variable and turning all of this off.

**Comment by marcofucci** [comment]
@danielquinn yes it's a bit of a pain in the ass but it's doable and has been done before. Just to expand on what you said:

- it's not a good idea to use redis instead of a db, just invest time in getting rid of any server-side storage completely. E.g. we can use encrypted signed cookies instead of a db.
- sites, contenttypes etc. are not needed and not so many other things depend on the db. In practice, you just can't use models and anything related to them

Just as a reference, you might find this useful: https://github.com/ministryofjustice/money-to-prisoners-common/tree/master/mtp_common/auth
